src/DynamoCoreWpf/Views/Preview/IWatch3DView.cs

```diff
@@ -0,0 +1,22 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using System.Windows.Input;
+using Autodesk.DesignScript.Interfaces;
+using HelixToolkit.Wpf.SharpDX;
+
+namespace Dynamo.Wpf.Views.Preview
+{
+    public interface IWatch3DView
+    {
+        Viewport3DX View { get; } 
```

Comment:
I don't like the idea of having a Helix specific class here so I might just go ahead and change this to a `IWatch3DViewModel` interface and expose mouse/keyboard events through that interface to be registered in the extension. That way we'll have a loose coupling with the View.